### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the Authentication Page P.R

image

### CODE REVIEW

**Review by purohitdheeraj — APPROVED** [review]
looking good i liked the UI in the preview image, it would be better if you host it on netlify

Suggestion would be to use px for less than 10px, at many places I noticed you have used rem for less than 10px.

Good to merge with this changes

File: Authentication/login.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+@import url("https://jatayui.netlify.app/comp-lib.css");
+/* component library */
+
+@import url("./../ecomPage.css");
+
+.login-page-conatiner
+{
+    display: flex;
+justify-content: center;
```
Comment (bharati-21) [inline Authentication/login.css]: Try to use a formatter like prettier to handle code formatting

File: forms.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    width: 80%;
+    height: 100%;
+}
+
+.text-field:focus {
+    border: 2px solid var(--primary-color);
+}
+
+/* 
+.input-other
```
Comment (bharati-21) [inline forms.css:78]: You can remove commented out code 👍🏽

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!
One suggestion is to use a code formatter to maintain consistencies with spacings.

File: forms.css (hunk 2 above)
Comment (raghav6202) [reply forms.css:78]: Thank you for the review Bharthi will look into the suggestions

**Pull request merged**